docs/source/migrating.rst

```diff
@@ -5,7 +5,7 @@
 Migrating from IPython
 ======================
 
-`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+`The Big Split <http://blog.jupyter.org/2015/04/15/the-big-split/>`__
```

Comment:
I think certain versions of Python on certain platforms actually have a non-functional SSL library. What version of Python are you using on what platform?